### ISSUE TITLE
Match color of lower sash to icon

### ISSUE THREAD

**Issue by RossBoylan** [microsoft/vscode#128814]
Issue Type: <b>Feature Request</b>

On Windows 10 the main VS code window has a purple bar at the bottom.  I would like it to be blue to match the icon on the task bar, which is blue.

The main problem is that the program icon for regular Visual Studio (not code) is purple.

So I keep thinking the purple icon on the task bar is what I need to click to focus on the VS Code window.

VS Code version: Code 1.58.2 (c3f126316369cd610563c75b1b1725e0679adfb3, 2021-07-14T22:10:15.214Z)
OS version: Windows_NT x64 10.0.17763
Restricted Mode: No

**Comment by IllusionMH** [comment]
Looks like you are opening separate files, but don't open folder (and have full workspace).
This is intentional to distinguish between to these two cases - folders vs separate files:
https://github.com/microsoft/vscode/blob/4b6444d73bb7ae2c9ba0c1bda37e574e8c232fd4/src/vs/workbench/common/theme.ts#L361-L371 

This is themable area so you can easily force it to have any color and make them same

```json
"workbench.colorCustomizations": {
     "statusBar.noFolderBackground": "#007ACC"
}
```

**Comment by misolori** [comment]
Yes, @IllusionMH is correct in that the purple status bar means you are not in a workspace/folder. If you open a project in VS Code it will turn blue.

**Comment by RossBoylan** [comment]
Yes, I am opening an individual file.

Given the desire to distinguish this from other cases, couldn’t some color that doesn’t match Visual Studio’s icon be used instead?
Thanks for the tip about how to customize it; I’ll try that locally.

Ross

**Comment by misolori** [comment]
Thanks for the feedback. Given that you can use an alternate theme or change this in your user settings I will close this issue out given that we've been using this color since the product launched. Will monitor to see if we get additional feedback.
